File: src/main/java/no/nav/fo/veilarbregistrering/bruker/PdlOppslagGateway.java
package no.nav.fo.veilarbregistrering.bruker;

import java.util.Optional;

public interface PdlOppslagGateway {

    Optional<Person> hentPerson(AktorId aktorid);

    Identer hentIdenter(Foedselsnummer fnr);

    Identer hentIdenter(AktorId aktorId);
}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/bruker/PdlOppslagGateway.kt
package no.nav.fo.veilarbregistrering.bruker

interface PdlOppslagGateway {
    fun hentPerson(aktorid: AktorId): Person?
    fun hentIdenter(fnr: Foedselsnummer, erSystemKontekst: Boolean = false): Identer
    fun hentIdenter(aktorId: AktorId): Identer
    fun hentGeografiskTilknytning(aktorId: AktorId): GeografiskTilknytning?
}
